lib/util/index.js

```diff
@@ -1,22 +1,12 @@
 import * as distance from './distance'
 import * as itinerary from './itinerary'
```

Comment:
Does itinerary need to exist anymore? What is unique in that file from otp-ui?

Reply:
There are some functions in itinerary that didn't originally get transferred over to OTP-ui.